### PR TITLE
Remove mistaken references to "projected" correlation function in docs

### PR DESCRIPTION
**Pull request by lgarrison**
Also update test instructions on RTD.  From #288.

### CODE REVIEW

**Comment by lgarrison** [comment]
@manodeep Looks like the MacOS jobs are failing, unrelated to this PR.  Maybe the `macos-latest` image got updated and broke something.  Could you take a look?

File: docs/source/install.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ The simplest way to install the latest release of the code is with pip. Before i
 
 .. code:: python
 
-          pip install Corrfunc
+    pip install Corrfunc
```
Comment (manodeep) [inline docs/source/install.rst:17]: Should this be `python -m pip install Corrfunc?`

**Comment by manodeep** [comment]
@lgarrison Looks good to me. Just had one comment about using our favoured `python -m pip` rather than `pip` directly. 

Semi-related, I did not get any email notification for the build failure. Did you?

File: docs/source/install.rst (hunk 1 above)
Comment (lgarrison) [reply docs/source/install.rst]: Agreed, thanks

**Comment by lgarrison** [comment]
Yes, I did get the emails, I guess only the PR author gets them. Thanks for the OSX fix; I'll merge as soon as the tests pass!

**Pull request merged**